configure.ac

```diff
@@ -285,6 +285,14 @@ GD_LIB_PKG_CHECK([LIBWEBP], [WEBP], [webp], [libwebp], [
   ])
 ])
 
+dnl Check for heif support.
+GD_LIB_PKG_CHECK([LIBHEIF], [HEIF], [heif], [libheif], [
```

Comment:
if there's a specific min version you want to enforce, you'd want to write `[libheif >= x.y.z]` here

Reply:
I'll target version [`1.7.0`](https://github.com/strukturag/libheif/releases/tag/v1.7.0) because it is the release that brought AVIF support (and the encoder options).